CRM/Financial/BAO/Payment.php

```diff
@@ -59,27 +59,36 @@ public static function create($params) {
 
     $isPaymentCompletesContribution = self::isPaymentCompletesContribution($params['contribution_id'], $params['total_amount']);
 
+    $whiteList = ['check_number', 'payment_processor_id', 'fee_amount', 'total_amount', 'contribution_id', 'net_amount', 'card_type_id', 'pan_truncation', 'trxn_result_code', 'payment_instrument_id', 'order_reference', 'trxn_id'];
```

Comment:
@eileenmcnaughton See comment on `order_reference` which is only available from 5.20 - though I don't think it will cause problems in this list.

Reply:
yep - removed